apps/andi/lib/andi/harvest/data_json_dataset_mapper.ex

```diff
@@ -6,7 +6,7 @@ defmodule Andi.Harvest.DataJsonToDataset do
 
   @scos_data_json_seed "1719bf64-38f5-40bf-9737-45e84f5c8419"
 
```

Comment:
I wish this file followed naming conventions.  Its module name does not match  its file  name